Sources/hostmgr/commands/vm/VMList.swift

```diff
@@ -79,6 +80,12 @@ struct VMListCommand: ParsableCommand {
         }
     }
 
+    private func printResumingVMs() throws {
+        try Parallels().lookupStoppingVMs().forEach {
+            printStatus(status: .resuming, virtualMachine: $0)
```

Comment:
😈 

```suggestion
            print(status: .resuming, virtualMachine: $0)
```

```suggestion
            printStatus(.resuming, virtualMachine: $0)
```

Reply:
Ah yeah nice one 😁 

Fixed in 1dcf573bd4db554262f6c89f7f2f15832a65cce8